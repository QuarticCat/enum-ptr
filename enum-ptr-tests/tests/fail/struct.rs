#![allow(dead_code)]

use enum_ptr::EnumPtr;

#[derive(EnumPtr)]
struct Foo<'a, 'b> {
    a: &'a i32,
    b: &'b i32,
}

fn main() {}
