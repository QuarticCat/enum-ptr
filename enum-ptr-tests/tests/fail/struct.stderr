error: EnumPtr only supports enums
 --> $DIR/struct.rs:6:8
  |
6 | struct Foo<'a, 'b> {
  |        ^^^
